test/repository_test.rb

```diff
@@ -82,6 +82,30 @@
             assert updated_user.updated_at > @updated_at
           end
         end
+
+        describe 'when undefined method for Class' do
+          before do
+            class Comment
+              attr_accessor :id
+              def initialize(attributes = {})
+              end
+            end
+
+            class CommentRepository
+              include Lotus::Repository
+            end
+
+            begin
```

Comment:
Can you please tell me why you wanted to raise exception here? 
Because I want to test case error. It will get the error if we don't include attributes when we create a repository.
But now, I got your opinion like that: I just test for case entity have  (updated_at or created_at ) and entity haven't (updated_at or created_at )